playbilling/src/main/java/com/google/androidbrowserhelper/playbilling/provider/MockBillingWrapper.java

```diff
@@ -21,21 +23,25 @@
 
     @Override
     public void connect() {
+        Log.d("Peter", "a");
```

Comment:
We probably want to remove those.

Reply:
:man_facepalming: 

I promise I do know how to use a debugger - I was just trying to figure out which parts of a test were taking so long.